lte/gateway/python/magma/subscriberdb/crypto/EC.py

```diff
@@ -0,0 +1,158 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+__all__ = ['X25519', 'ECDH_SECP256R1', 'KDF']
+"""
+
+# this is a wrapper around the cryptography Python librar and its elliptic curve
```

Comment:
```suggestion
# this is a wrapper around the cryptography Python library and its elliptic curve
```

Reply:
Done

Reply:
Maybe you haven't pushed your changes yet? All of these look unaddressed to me.

Reply:
Somehow i missed to amend this change. Done now.